web-api/migrations/00018-eligible-for-trial-docket-number.test.js

```diff
@@ -0,0 +1,96 @@
+const { forAllRecords } = require('./utilities');
+const { up } = require('./00018-eligible-for-trial-catalog-docket-number');
```

Comment:
This test file name is missing the word "catalog" in it, per the testee filename.

Reply:
Good catch!

Reply:
Updated!